draft-ietf-tls-rfc8446bis.md

```diff
@@ -3100,66 +3100,61 @@ in TLS 1.2 and below.
+
+- The "server_name" {{RFC6066}} extension is used to guide certificate
+  selection. As servers MAY require the presence of the "server_name" extension,
+  clients SHOULD send this extension when the server is identified by name.
+
+All certificates provided by the sender MUST be signed by a
+signature algorithm advertised by the peer, if it is able to provide such
 a chain (see {{signature-algorithms}}).
 Certificates that are self-signed
 or certificates that are expected to be trust anchors are not validated as
 part of the chain and therefore MAY be signed with any algorithm.
 
 If the server cannot produce a certificate chain that is signed only via the
 indicated supported algorithms, then it SHOULD continue the handshake by sending
-the client a certificate chain of its choice that may include algorithms
-that are not known to be supported by the client.
+a certificate chain of its choice that may include algorithms that are not known
+to be supported by the peer.
 This fallback chain SHOULD NOT use the deprecated SHA-1 hash algorithm in general,
-but MAY do so if the client's advertisement permits it,
-and MUST NOT do so otherwise.
+but MAY do so if the peer's advertisement permits it,
+and MUST NOT do so otherwise. Clients MAY send a fallback chain as above, or
+continue the handshake anonymously.
 
-If the client cannot construct an acceptable chain using the provided
+If the receiver cannot construct an acceptable chain using the provided
 certificates and decides to abort the handshake, then it MUST abort the
 handshake with an appropriate certificate-related alert (by default,
 "unsupported_certificate"; see {{error-alerts}} for more information).
 
-If the server has multiple certificates, it chooses one of them based on the
+If the sender has multiple certificates, it chooses one of them based on the
 above-mentioned criteria (in addition to other criteria, such as transport-layer endpoint, local configuration, and preferences).
-
-#### Client Certificate Selection
-
-The following rules apply to certificates sent by the client:
-
-- The certificate type MUST be X.509v3 {{RFC5280}}, unless explicitly negotiated
-  otherwise (e.g., {{RFC7250}}).
-
-- If the "certificate_authorities" extension in the CertificateRequest
-  message was present, at least one of the certificates in the certificate
-  chain SHOULD be issued by one of the listed CAs.
-
-- The certificates MUST be signed using an acceptable signature
-  algorithm, as described in {{certificate-request}}.  Note that this
-  relaxes the constraints on certificate-signing algorithms found in
-  prior versions of TLS.
-
-- If the CertificateRequest message contained a non-empty "oid_filters"
-  extension, the end-entity certificate MUST match the extension OIDs
-  that are recognized by the client, as described in {{oid-filters}}.
+Future TLS extensions may extend this criteria to further guide this selection
```

Comment:
This is always the case; I see no reason to spell it out.

Reply:
Removed.